### PR TITLE
sl docs improvements

### PR DESCRIPTION
**Pull request by mirnawong1**
edits to sl docs per [slack thread](https://dbt-labs.slack.com/archives/C04437Z6M5F/p1668657229019019)

## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`
- [ ] [Run link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

**Review by cafzal — DISMISSED** [review]
- In the first section: "The dbt Semantic Layer provides the flexibility to define metrics on top of your existing models then query those metrics and models in your analysis tools of choice."
- Under product architecture for "dbt project" - description: "Define models and metrics in dbt Core."
- Under what makes the SL different, the new line: "You can also use the dbt Semantic Layer to query models and use macros."

File: website/docs/docs/use-dbt-semantic-layer/dbt-semantic-layer.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ sidebar_label: "dbt Semantic Layer"
 
 <Snippet src="sl-public-preview-banner" />
 
-The dbt Semantic Layer allows data teams to centrally define essential business metrics like `revenue`, `customer`, and `churn` in the modeling layer (your dbt project) for consistent self-service within downstream data tools like BI and metadata management solutions.
+The dbt Semantic Layer allows data teams to centrally define essential business metrics like `revenue`, `customer`, and `churn` in the modeling layer (your dbt project) for consistent self-service within downstream data tools like BI and metadata management solutions.  With the dbt Semantic Layer, you have the flexibility to define metrics from scratch or define metrics using existing models.
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/dbt-semantic-layer.md]:
```suggestion
The dbt Semantic Layer allows data teams to centrally define essential business metrics like `revenue`, `customer`, and `churn` in the modeling layer (your dbt project) for consistent self-service within downstream data tools like BI and metadata management solutions.  The dbt Semantic Layer provides the flexibility to define metrics on top of your existing models and then query those metrics and models in your analysis tools of choice.
```

File: website/docs/docs/use-dbt-semantic-layer/dbt-semantic-layer.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ The dbt Semantic Layer product architecture includes four primary components:
 
 | Components | Information | Developer plans | Team plans | Enterprise plans | License |
 | --- | --- | :---: | :---: | :---: | --- |
-| **[dbt metrics](/docs/build/metrics)** | Allows you to define metrics in dbt Core. | ✅ | ✅ |  ✅  | Open source, Core |
+| **[dbt project](/docs/build/metrics)** | Allows you to define metrics in dbt Core. | ✅ | ✅ |  ✅  | Open source, Core |
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/dbt-semantic-layer.md]:
```suggestion
| **[dbt project](/docs/build/metrics)** | Define models and metrics in dbt Core. | ✅ | ✅ |  ✅  | Open source, Core |
```

File: website/docs/docs/use-dbt-semantic-layer/dbt-semantic-layer.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ The dbt Semantic Layer has four main parts:
 
 ### What makes the dbt Semantic Layer different?
 
-The dbt Semantic Layer reduces code duplication and inconsistency regarding your business metrics. By moving metric definitions out of the BI layer and into the modeling layer, data teams can feel confident that different business units are working from the same metric definitions, regardless of their tool of choice. If a metric definition changes in dbt, it’s refreshed everywhere it’s invoked and creates consistency across all applications.
+The dbt Semantic Layer reduces code duplication and inconsistency regarding your business metrics. By moving metric definitions out of the BI layer and into the modeling layer, data teams can feel confident that different business units are working from the same metric definitions, regardless of their tool of choice. If a metric definition changes in dbt, it’s refreshed everywhere it’s invoked and creates consistency across all applications. You can also use the dbt Semantic Layer to also query models and macros.
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/dbt-semantic-layer.md]:
```suggestion
The dbt Semantic Layer reduces code duplication and inconsistency regarding your business metrics. By moving metric definitions out of the BI layer and into the modeling layer, data teams can feel confident that different business units are working from the same metric definitions, regardless of their tool of choice. If a metric definition changes in dbt, it’s refreshed everywhere it’s invoked and creates consistency across all applications. You can also use the dbt Semantic Layer to query models and use macros.
```

**Comment by mirnawong1** [comment]
thanks @cafzal - made those changes and let me know what you think? if you're good with them, I'll ask for a docs team review + approval. thanks!

File: website/docs/docs/use-dbt-semantic-layer/dbt-semantic-layer.md [hunk 4]
```diff
@@ -7,7 +7,7 @@ sidebar_label: "dbt Semantic Layer"
 
 <Snippet src="sl-public-preview-banner" />
 
-The dbt Semantic Layer allows data teams to centrally define essential business metrics like `revenue`, `customer`, and `churn` in the modeling layer (your dbt project) for consistent self-service within downstream data tools like BI and metadata management solutions.
+The dbt Semantic Layer allows data teams to centrally define essential business metrics like `revenue`, `customer`, and `churn` in the modeling layer (your dbt project) for consistent self-service within downstream data tools like BI and metadata management solutions.  The dbt Semantic Layer provides the flexibility to define metrics on top of your existing models and then query those metrics and models in your analysis tools of choice.
```
Comment (matthewshaver) [inline website/docs/docs/use-dbt-semantic-layer/dbt-semantic-layer.md]:
```suggestion
The dbt Semantic Layer allows data teams to centrally define essential business metrics like `revenue`, `customer`, and `churn` in the modeling layer (your dbt project) for consistent self-service within downstream data tools like BI and metadata management solutions. The dbt Semantic Layer provides the flexibility to define metrics on top of your existing models and then query those metrics and models in your analysis tools of choice.
```

**Review by matthewshaver — APPROVED** [review]

**Pull request merged**